src/map/script.c

```diff
@@ -11775,6 +11775,13 @@ static BUILDIN(getunits)
 		const char *mapname = script_getstr(st, 5);
 		int16 m = map->mapname2mapid(mapname);
 
+		if (m == -1) {
```

Comment:
I'd prefer `INDEX_NOT_FOUND` instead of `-1`, but that's up to you.

Reply:
You are correct. will change map_mapname2mapid and map_mapindex2mapid functions too, so that it will be in sync

Reply:
@Kenpachi2k13 I will do it in new PR, as the purpose of that would be different, and it would be better to change at all places where mapname2mapid is called.

Reply:
:+1: